File: src/main/java/com/gmail/borlandlp/minigamesdtools/gui/hotbar/items/HotbarItemCreatorHub.java
package com.gmail.borlandlp.minigamesdtools.gui.hotbar.items;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class HotbarItemCreatorHub extends CreatorHub {
    public SlotItem createHotbarItem(String ID, AbstractDataProvider dataProvider) throws Exception {
        return (SlotItem) this.create(ID, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/gui/hotbar/items/HotbarItemCreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.gui.hotbar.items

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class HotbarItemCreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): SlotItem {
        return super.create(itemID, dataProvider) as SlotItem
    }
}